src/index.js

```diff
@@ -665,7 +665,9 @@ export async function refetchAllQueries({
         return query.fetch({ force })
       }
     })
-  )
+  ).catch(error => { 
+    console.error(error.message)
+  });
 }
```

Comment:
This is great! Although, what this does now, is makes it so that if you import and use the `refetchAllQueries` function intentionally, you can't detect errors :(

Reply:
This block however:

https://github.com/tannerlinsley/react-query/pull/37/files#diff-1fdf421c05c1140f6d71444ea2b27638R13-R15

is where the window-focus fetching happens. You may have better luck doing something similar there :)

Reply:
https://github.com/tannerlinsley/react-query/blob/5194bbe7dee13008d08c990eed98b1a3bd8844ae/src/index.js#L13-L15

Reply:
I now catch the errors only for the usage of this method instead inside the method, thanks!